index.js

```diff
@@ -32,6 +39,22 @@ module.exports = {
   env: {
     browser: true,
     node: true,
+    // we are using optional chaining and nullish coalescing
+    es2020: true,
+  },
+  globals: {
+    // these globals help IDEs from displaying these as undefined
+    localStorage: true,
+    process: false,
+    Promise: true,
+  },
+  settings: {
+    // the src folder can be used as the resource root for imports
+    'import/resolver': {
+      node: {
+        paths: ['src'],
```

Comment:
Would this change require us to update the way we use webpack? (eg `{ resolve: { modules: [path.resolve(__dirname, 'src'), 'node_modules'] } }`)

Reply:
Nope.